osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/config/messaging/OutboundLogItemRequestsMessagingConfig.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.core.application.config.messaging;
+
+import javax.net.ssl.SSLException;
+
+import org.apache.activemq.pool.PooledConnectionFactory;
+import org.opensmartgridplatform.shared.application.config.jms.JmsConfigurationNames;
+import org.opensmartgridplatform.shared.application.config.messaging.JmsConfiguration;
+import org.opensmartgridplatform.shared.application.config.messaging.JmsConfigurationFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.context.annotation.PropertySource'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
I agree.